docs/getting-started-scalardb.md

```diff
@@ -0,0 +1,344 @@
+
+We need to start the Cassandra container as backend storage of the Scalar DB Server on the same network of the minikube, so you need to be started the minikube with docker driver.
+
+1. Start minikube with docker driver.
+   ```console
+   $ minikube start --driver=docker
+   ```
+
+1. Check the status of the minikube and pods.
+   ```console
+   $ kubectl get pod -A
+   NAMESPACE     NAME                               READY   STATUS    RESTARTS        AGE
+   kube-system   coredns-64897985d-6jw6c            1/1     Running   0               3m27s
+   kube-system   etcd-minikube                      1/1     Running   27              3m38s
+   kube-system   kube-apiserver-minikube            1/1     Running   27              3m38s
+   kube-system   kube-controller-manager-minikube   1/1     Running   27              3m38s
+   kube-system   kube-proxy-5vcnc                   1/1     Running   0               3m27s
+   kube-system   kube-scheduler-minikube            1/1     Running   27              3m38s
+   kube-system   storage-provisioner                1/1     Running   1 (2m56s ago)   3m37s
+   ```
+   If the minikube started properly, you can see some pods are `Running` in the kube-system namespace.
+
+## Step 3. Start Cassandra container
+
+In this guide, we use Apache Cassandra as backend storage of Scalar DB Server, it will be started on the same network of Scalar DB Server (Pod on minikube) to enable proper communication.
+
+1. Start Cassandra container on the Docker Network `minikube`.
+   ```console
+   $ docker run --name cassandra-scalardb --network minikube -d cassandra:3.11
+   ```
+   * Note: 
+       * The Docker Network `minikube` was created by `minikube start --driver=docker` command that we ran in Step 2.
+       * You can see the Scalar DB-supported Cassandra versions (tag) in [this document](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md).
+       * If you want to create a schema for Scalar DB by running Schema Loader from localhost, you need to expose the container's 9042 port as follows.
+         ```console
+         $ docker run --name cassandra-scalardb --network minikube -p 127.0.0.1:9042:9042 -d cassandra:3.11
+         ```
+
+1. Check the Cassandra container is running.
+   ```console
+   $ docker ps -f name=cassandra-scalardb
+   CONTAINER ID   IMAGE            COMMAND                  CREATED         STATUS         PORTS                                         NAMES
+   6dceab0007c8   cassandra:3.11   "docker-entrypoint.s…"   2 minutes ago   Up 2 minutes   7000-7001/tcp, 7199/tcp, 9042/tcp, 9160/tcp   cassandra-scalardb
+   ```
+
+1. Check the status of Cassandra.
+   ```console
+   $ docker exec -t cassandra-scalardb cqlsh -e "show version"
+   [cqlsh 5.0.1 | Cassandra 3.11.11 | CQL spec 3.4.4 | Native protocol v4]
+   ```
+   It may take some time to start Cassandra in the container. So, if this command return error, wait a moment and then re-run it.
```

Comment:
```suggestion
   It may take a while to start Cassandra in the container. So, if this command returns an error, wait a moment and then re-run it.
```